parachains/runtimes/assets/asset-hub-kusama/src/lib.rs

```diff
@@ -216,7 +216,7 @@ impl pallet_balances::Config for Runtime {
 	type ReserveIdentifier = [u8; 8];
 	type RuntimeHoldReason = RuntimeHoldReason;
 	type FreezeIdentifier = ();
-	type MaxHolds = ConstU32<0>;
+	type MaxHolds = ConstU32<1>;
```

Comment:
```suggestion
	type MaxHolds = ConstU32<1>; // the hold is from `NftFractionalization`
```

Reply:
like this?

Reply:
I would do more like:
```suggestion
	// We allow each account to have holds on it from:
	//   - `NftFractionalization`: 1
	type MaxHolds = ConstU32<1>;
```